DockerHubfile

```diff
@@ -0,0 +1,32 @@
+#### Create Jar 
+FROM gradle:jdk8 as builder 
+ 
+COPY . /home/gradle/app 
+USER root
+RUN chown -R gradle:gradle /home/gradle/app
+
+USER gradle
+WORKDIR /home/gradle/app 
+ 
+RUN mkdir -p api/src/main/resources/META-INF 
+RUN echo "build.version=$(gradle -q printVersion)" > api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.number=${BUILD_NUMBER:=docker}" >> api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.commit=$(git rev-parse HEAD)" >> api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.date=$(date)" >> api/src/main/resources/META-INF/build-info.properties 
```

Comment:
can be minimised in to single image exec with command like
```dockerfile
RUN mkdir .... \
  && echo ... \
  && etc
```

not that it matters much